source/validate_id.cpp

```diff
@@ -454,7 +454,7 @@ bool idUsage::isValid<SpvOpConstantComposite>(const spv_instruction_t* inst,
           return false;
         }
         auto constituentResultType =
-            usedefs_.FindDef(constituent.second.words[1]);
+            usedefs_.FindDef(constituent.second.type_id);
         if (!constituentResultType.first ||
             componentType.second.opcode !=
                 constituentResultType.second.opcode) {
```

Comment:
I think you missed a words[1] instance on line 497.  For vector member of OpTypeMatrix.
Test case would be a matrix with a label as argument, for example.